src/components/tools/table-tool/data-table.css

```diff
@@ -31,6 +31,14 @@
   border-bottom: 1px dotted silver;
 }
 
+.ag-theme-fresh .ag-row-selected {
+  background-color: #bde2e5;
```

Comment:
Out of curiosity, where does this color value come from? Do we need to specify the default behavior here?

Reply:
Actually, I couldn't figure out exactly where ag-grid defines the selected-row background color. So, to keep the original behavior intact (when the table is not linked to a geometry) I just forced it to the hardcoded value it originally displayed. Eventually, I would want to define a whole set of grid-colors to get rid of all the magic numbers.

And, as you observe, this could be removed and the default behavior would not be different -- but I found it a bit easier to understand to hard-code it.

Reply:
I don't feel particularly strongly about it, but my inclination would be to remove it. My thinking is that:
1. It is contrary to the (worthy) goal you set out in your introduction of being minimally invasive.
2. There is probably some ag-grid API somewhere for specifying the default row selection color. Somewhere down the road someone will try to use that API and then be confused about why it doesn't work as expected.

That said, it's clearly a judgment call and I'm fine merging it either way.

Reply:
You got it.